tasks/arrays/arrays.js

```diff
@@ -1,62 +1,81 @@
-/**
- * 1. Write a method to reverse a string; Function should return the string 'This is not a string'
- * if a number is passed
-* @param {string}
-*/
-function reverseString(str) {
-}
-
 /**
  * return an array of characters names
+ * @param {Array} chars
+ * @return {Array} - char names
  */
-function getCharactersNames(chars) {
+function getCharactersNames(chars, ) {
+  let result = chars.map(el => el.name);
+  return result;
 }
 
 /**
- * print (console.log) ids of all characters
- */
+* print (console.log) ids of all characters
+* @param {Array} chars
+*/
 function printCharacterNames(chars) {
+ let result =  chars.forEach(obj => console.log(obj.name));
+return result;
 }
 
 /**
- * return an array of non-human characters
- */
+* return an array of non-human characters
+* @param {Array} chars
+* @return {Array} - non human characters
+*/
 function getNonHumanCharacters(chars) {
+  let result = chars.filter(obj => {
+      return obj.species !== 'Human'
+    });
+    return result;
 }
 
 /**
- * return info about character named 'Jerry Smith'
- */
+* return info about character named 'Jerry Smith'
+* @param {Array} chars
+* @return {Object} - Jerry object
+*/
 function getJerryInfo(chars) {
+let result = chars.find(char => char.name === 'Jerry Smith');
+    return result;
 }
 
+
 /**
- * check if all characters are human. return true if all, false if not
- */
+* check if all characters are human. return true if all, false if not
+* @param {Array} chars
+* @return {boolean}
+*/
 function isAllHuman(chars) {
-}
+  let result = chars.every(o => o.species == 'Human') ? true : false;
```

Comment:
This is superfluously, just leave:
let result = chars.every(o => o.species == 'Human');
Every() will return TRUE if all of the statements are true otherwise, it will be false (exactly as you need).